src/SmartComponents/HybridInventoryTabs/ImmutableDevices.js

```diff
@@ -135,7 +132,12 @@ const ImmutableDevices = ({ rule, pathway, selectedTags }) => {
       }}
       key="inventory"
       customFilters={{
-        advisorFilters: filters,
+        advisorFilters: {
+          ...filters,
+          ...(isEdgeParityEnabled
+            ? { 'filter[system_profile][host_type]': 'edge' }
+            : {}),
+        },
```

Comment:
nit-pick: do you mind extracting this to a function? and we could call it like "buildAdvisorEdgeFilters" to better illustrate that this parameter is (and has to be) wired into the filters.